jenkins/pipelines/releng/promote-a-build.Jenkinsfile

```diff
@@ -0,0 +1,66 @@
+pipeline {
+  agent {
+    kubernetes {
+      label 'cdt-upload-agent-pod'
+      yaml """
```

Comment:
@Kummallinen I struggled for a while to get this into a pod template and ultimately gave up. I will try again when I move next jobs over and if it still fails I may ask you for some advice here.

Reply:
Should at least change the label, having different templates with the same label seems to confuse jenkins when you have jobs for both templates. You can have a job allocated to the wrong agent.